packages/core/src/commands/config/db.ts

```diff
@@ -0,0 +1,71 @@
+import { flags } from "@oclif/command";
+import { hasSomeProperty } from "@arkecosystem/core-utils";
+import { existsSync } from "fs-extra";
+import { CommandFlags, EnvironmentVars } from "../../types";
+import { BaseCommand } from "../command";
+import { updateEnvironmentVariables } from "../../utils";
+
+export class DatabaseCommand extends BaseCommand {
+    public static description: string = "Update the Database configuration";
+
+    public static examples: string[] = [
+        `Set the database host
+$ ark config:db --host=localhost
+`,
+        `Set the name of the database user
+$ ark config:db --username=ark
+`,
+        `Set the database name
+$ ark config:db --database=ark_mainnet
+`,
+        `Set the database password
+$ ark config:db --password=password
+`,
+    ];
+
+    private static readonly validFlags: string[] = ["host", "username", "database", "password"];
```

Comment:
Any reason you didn't include a `port` flag?

Reply:
I went by the vars laid out in https://docs.ark.io/guidebook/core/configuration.html#environment-configuration, i'll check up on the port envvar.

Reply:
I'll add it here and to the docs asap.